docs/faq/onboarding/extension.md

```diff
@@ -40,17 +40,59 @@ The Universal Profile Browser Extension currently does not feature a way to impo
 
 :::
 
-## How to check if 2FA is set up correctly?
+## Can I add 2FA recovery to an existing Universal Profile?
 
-You can check if the recovery is set up correctly using the Universal Profile Browser Extension. If you visit the Controller Menu, there need to be at least 3 Controllers:
+At the current state (11th January 2024), the 2FA recovery method from LUKSO can only be added during the creation of a profile on [UniversalProfile.cloud](https://universalprofile.cloud/).
+
+## How to check if 2FA recovery is set up correctly?
+
+If you ever want to recover your profile, a fresh controller key will be added to this profile from a new extension setup. You must verify multiple steps to check if this new controller key can be added to your Universal Profile by the recovery service:
+
+### 1. Verify the recovery controller in your extension
+
+If you visit the controller menu in the Universal Profile Browser Extension, there need to be at least three controllers:
+
+- The Universal Profile Browser Extension (used for profile interactions)
+- The Universal Receiver (used to write asset updates on the profile)
+- The Universal Profile Recovery (used as a backup for the profile)
+
+To confirm that the permissions of the recovery controller were adequately set up, click on the controller of the Universal Profile Recovery and verify that the **Add Controller** permission is enabled.
 
 ![2FA Controller](../../../static/img/faq/2fa-controller.png)
 
-- Your Universal Profile Browser Extension's Key (used for profile interactions)
-- Your Universal Receiver (used to write asset updates on the profile)
-- The Universal Recovery Key (used as a backup for the profile)
+:::info Controller Names
+
+The names of the controllers can vary, depending on the time of your Universal Profile setup.
+
+:::
+
+### 2. Verify the recovery controller on the profile explorer
+
+Next, we must check if the recovery was successfully added via the LUKSO Relay Service. **Copy the address of your Universal Profile Recovery** from the controller menu of the Universal Profile Extension and open [UniversalProfile.cloud](https://universalprofile.cloud/?network=mainnet). On the profile explorer, **input the address** of the Universal Profile Recovery **into the search bar**. If it was successfully added, it will show a recovery profile.
```

Comment:
Suggest doing a numbered list here:

```suggestion
Next, we must check if the recovery was successfully added via the LUKSO Relay Service. 

1. **Copy the address of your Universal Profile Recovery** from the controller menu of the Universal Profile Extension
2. open [UniversalProfile.cloud](https://universalprofile.cloud/?network=mainnet). 
3. On the profile explorer, **input the address** of the Universal Profile Recovery **into the search bar**. 

If it was successfully added, it will show a recovery profile.
```

Reply:
I am not sure to understand. How does finding the recovery UP in the universalprofile.cloud explorer gives you the guarantee that this recovery UP has been setup for your UP?

It should rather be something to check on the dashboard of my.universalprofile.cloud no?

Reply:
It is not registered on the relayer if it has not been added to the profile. At least that's what I was told.